vendor/pischem/caddy/Makefile

```diff
@@ -0,0 +1,8 @@
+PLUGIN_NAME=            caddy
+PLUGIN_VERSION=         1.5.0
+PLUGIN_DEPENDS=         caddy
+PLUGIN_COMMENT=         Easy to configure Reverse Proxy based on Caddy with Automatic HTTPS and Dynamic DNS	 
+PLUGIN_MAINTAINER=      cedrik@pischem.com
+PLUGIN_WWW=             https://github.com/Monviech/os-caddy-plugin
```

Comment:
You can point this to a documentation not directly included, but in general we would want to avoid out of tree work and references to avoid confusion of unseres about development and stable state between different repos.